homeassistant/components/knx/config_flow.py

```diff
@@ -35,6 +47,12 @@ class FlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     _gateway_ip: str = ""
     _gateway_port: int = 3675
```

Comment:
maybe move default port to const.py or import from xknx

Reply:
adjusted